validation/superchain-version_test.go

```diff
@@ -16,9 +16,52 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
+var isSemverAcceptable = func(desired, actual string) bool {
+	return desired == actual
+}
+
+// TestSuperchainWideContractVersions will check that
+//   - for each superchain
+//   - for each declared superchain-wide contract "Foo" : version entry in the corresponding superchain's semver.yaml
+//   - the superchain has a contract Foo deployed (at an address declared in superchain.yaml) at the same version
+//
+// Actual semvers are
+// read from the L1 chain RPC provider for the superchain in question.
```

Comment:
I'm not sure if we really want to keep this godoc here (same for the other test in this file). 

* This function will not be imported by another package, so no "API" to document. 
* We risk it getting out of sync with the implementation.
* The code should be clear enough to document itself.
* godoc references underlying source data files (when in truth it operates only on Go types)

Reply:
[4b3a478](https://github.com/ethereum-optimism/superchain-registry/pull/73/commits/4b3a4784f1aef72188492c1657c618c2f95f23b5)